<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findBook</class>
 <widget class="QDialog" name="findBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1315</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1315</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>我要找书</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#findBook{border-image: url(:/pic/background_blurred.jpg);}
QLineEdit {
	border: 2px solid rgba(180, 180,180,0.3);
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: rgba(204, 204, 204,0.2); /* 背景颜色 */
	color: rgb(50,50,50); /* 文本颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	/*font-size: 20pt; /* 文本字体大小 */
	letter-spacing: 1px;
}

QLineEdit:focus{ /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid rgba(160, 160,160,0.8);
	border-radius: 6px;
	background-color:  rgba(180, 180, 180,0.2);
	color: rgb(40,40,40);
	selection-background-color: rgba(142,210,255,0.6); /* 选中文本的背景颜色 */
	selection-color: rgb(40,40,40);
}

QLineEdit:disabled{
	border: 2px solid rgba(214, 217,219,1);
	border-radius: 8px;
	background-color:  rgba(228, 233, 236,0.5);
	color: rgb(40,40,40)
}

QPushButton {background-color: rgba(160,160, 160,0.2);padding: 5px 6px 5px 6px;font: 27px 'Microsoft YaHei';letter-spacing: 2px;border: transparent;border-radius: 6px;}
QPushButton:pressed:hover{background-color: rgba(153, 153, 153,0.5);}
QPushButton:hover {background-color: rgba(230, 230, 230,0.4);border: 2px solid rgba(180,180,180,0.8)}
QPushButton:disabled {background-color: rgba(204, 204, 204,0.3);color: rgb(120, 120, 120);}
QLabel{letter-spacing: 1px;}

QTextEdit {
	border: 2px solid rgba(180, 180,180,0.3);
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: rgba(204, 204, 204,0.2); /* 背景颜色 */
	color: rgb(40,40,40); /* 文本颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	/*font-size: 20pt; /* 文本字体大小 */
	letter-spacing: 1px;
}

QTextEdit:focus{ /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid rgba(160, 160,160,0.8);
	border-radius: 6px;
	background-color:  rgba(180, 180, 180,0.2);
	color: rgb(40,40,40);
	selection-background-color: rgba(142,210,255,0.6); /* 选中文本的背景颜色 */
	selection-color: rgb(40,40,40);
}

QScrollBar:vertical
{
       border: none;
       background: transparent;
       width: 8px;
}
 
QScrollBar::handle:vertical
{
      background-color: rgba(150,150,150,1);
      min-height: 30px;
	  border-radius:3px;
	  width: 6px;
}
 
QScrollBar::add-line:vertical
{
       border: none;
       height: 0px;
}
 
QScrollBar::sub-line:vertical
{
       border: none;
       height: 0px;
}
 
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{	
		background:transparent;
      	border-radius:3px;
}</string>
  </property>
  <widget class="QLineEdit" name="words_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>485</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="press_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>195</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="clc_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>77</y>
     <width>221</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="bookDesc_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>137</y>
     <width>291</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:14pt; font-weight:400; font-style:normal; letter-spacing:1px;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>79</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>137</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="backLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(245, 245, 245,0.8);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>197</y>
     <width>96</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版社：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="shelfNum_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>137</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Tip">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>350</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 0, 0, 0.6);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>81</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书架号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_Btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>477</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>369</y>
     <width>241</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pressDate_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>253</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pressPlace_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>311</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>743</x>
     <y>373</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>元</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="bookName_Edit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>9</y>
     <width>1291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>请输入书名</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>369</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>价格：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>138</y>
     <width>78</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>485</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="isBorrowed_Label">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>290</y>
     <width>112</width>
     <height>48</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>83</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>313</y>
     <width>96</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版地：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="pages_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>427</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>页数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>255</y>
     <width>120</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版日期：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分类：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="find_Btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>420</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>带我去</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>350</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{background-color: rgba(235, 235, 235, 0.2);outline:none;}
QListWidget::item{height: 114px;}
QListWidget::item:hover {background-color: rgba(195,195,195,0.5);}
QListWidget::item:selected {background-color: rgba(175,175,175,0.5);border-left: 5px solid rgb(44, 181, 255);}

QScrollBar:vertical
{
       border: none;
       background: transparent;
       width: 6px;
}
 
QScrollBar::handle:vertical
{
      background-color: rgba(170,170,170,1);
      min-height: 30px;
	  border-radius:3px;
	  width: 6px;
}
 
QScrollBar::add-line:vertical
{
       border: none;
       height: 0px;
}
 
QScrollBar::sub-line:vertical
{
       border: none;
       height: 0px;
}
 
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{	
		background:transparent;
      	border-radius:3px;
}</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>backLabel</zorder>
  <zorder>words_Edit</zorder>
  <zorder>press_Edit</zorder>
  <zorder>clc_Edit</zorder>
  <zorder>bookDesc_Edit</zorder>
  <zorder>name_Edit</zorder>
  <zorder>author_Edit</zorder>
  <zorder>label_3</zorder>
  <zorder>shelfNum_Edit</zorder>
  <zorder>Tip</zorder>
  <zorder>label_11</zorder>
  <zorder>quit_Btn</zorder>
  <zorder>price_Edit</zorder>
  <zorder>pressDate_Edit</zorder>
  <zorder>pressPlace_Edit</zorder>
  <zorder>label_10</zorder>
  <zorder>bookName_Edit</zorder>
  <zorder>label_6</zorder>
  <zorder>label_2</zorder>
  <zorder>label_8</zorder>
  <zorder>isBorrowed_Label</zorder>
  <zorder>label</zorder>
  <zorder>label_5</zorder>
  <zorder>pages_Edit</zorder>
  <zorder>label_7</zorder>
  <zorder>label_4</zorder>
  <zorder>label_9</zorder>
  <zorder>find_Btn</zorder>
  <zorder>listWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
